tests/resources/VCH-Util.robot

```diff
@@ -412,6 +415,8 @@ Install VIC with version to Test Server
     ${rc}  ${output}=  Run And Return Rc And Output  tar zxvf vic.tar.gz
     Set Environment Variable  TEST_TIMEOUT  20m0s
     Install VIC Appliance To Test Server  vic-machine=./vic/vic-machine-linux  appliance-iso=./vic/appliance.iso  bootstrap-iso=./vic/bootstrap.iso  certs=${false}  vol=default ${insecureregistry}
+
+    Clean VMOMI Sessions  vic-machine  # old versions of vic-machine may leak sessions, confusing other tests
```

Comment:
This isn't cleaning out an old build, it's cleaning up a VMOMI session from a version of vic-machine that contains the bug this PR fixes. If we added this to the install / delete keywords, then the actual test would be pointless because we would clean out all of the leaked vic-machine sessions and then check to see if there are any leaked sessions.